src/templates/subscription/bar.html

```diff
@@ -0,0 +1,9 @@
+<div class="pf-widget-body">
+</div>
+<a class="pf-widget-close">&times;</a>
+<div class="pf-bar-content">
+	<p class="pf-widget-message"></p>
+	<form>
+		<input name="email" type="email" required><input type="submit" class="pf-widget-btn pf-widget-ok" />
```

Comment:
This is not on you, and may be something for a separate pr, but I find it weird that some of the common templates have inconsistencies, like this is the only subscription template that uses a `input` vs. a `button` for the confirm button.